framework/db/CDbConnection.php

```diff
@@ -575,11 +575,12 @@ public function quoteValue($str)
 		if(is_int($str) || is_float($str))
 			return $str;
 
-		$this->setActive(true);
-		if(($value=$this->_pdo->quote($str))!==false)
-			return $value;
-		else  // the driver doesn't support quote (e.g. oci)
-			return "'" . addcslashes(str_replace("'", "''", $str), "\000\n\r\\\032") . "'";
+        $this->setActive(true);
```

Comment:
Formatting should use tabs. Need to fix that before merge.

Reply:
@samdark fixed. used tabs now